tests/jenkins/TestCreateUploadTestReportManifest.groovy

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package jenkins.tests
+
+import org.junit.Test
+import static com.lesfurets.jenkins.unit.MethodCall.callArgsToString
+import static org.hamcrest.CoreMatchers.hasItems
+import static org.hamcrest.MatcherAssert.assertThat
+
+
+class TestCreateUploadTestReportManifest extends BuildPipelineTest {
+
+    @Test
+    public void TestCreateUploadTestReportManifest() {
+        this.registerLibTester(new CreateUploadTestReportManifestLibTester(
+                'tests/data/opensearch-1.3.0-test.yml',
+                'tests/data/opensearch-1.3.0-build.yml',
+                'tests/data/opensearch-dashboards-build-1.3.0.yml',
+                '1234',
+                'integ-test',
+                '',
+            )
+        )
+        super.setUp()
+        super.testPipeline("tests/jenkins/jobs/CreateUploadTestReportManifest_Jenkinsfile")
+        assertThat(getShellCommands('sh', 'report.sh'), hasItems('./report.sh tests/data/opensearch-1.3.0-test.yml --artifact-paths opensearch=https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/1.3.0/c3ff7a232d25403fa8cc14c97799c323/linux/x64/tar --test-run-id 1234 --test-type integ-test --base-path null/null/1.3.0/c3ff7a232d25403fa8cc14c97799c323/linux/x64/tar '))
+        assertThat(getShellCommands('sh', 'report.sh'), hasItems('./report.sh tests/data/opensearch-1.3.0-test.yml --artifact-paths opensearch=https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/1.3.0/c3ff7a232d25403fa8cc14c97799c323/linux/x64/tar opensearch-dashboards=https://ci.opensearch.org/ci/dbc/distribution-build-opensearch-dashboards/1.3.0/25b38c278cdd45efa583765d8ba76346/linux/x64/tar --test-run-id 1234 --test-type integ-test --base-path null/null/1.3.0/c3ff7a232d25403fa8cc14c97799c323/linux/x64/tar '))
```

Comment:
This should have opensearch-dashboards test manifest. Also base path is going null here: 
```--base-path null/null/1.3.0/c3ff7a232d25403fa8cc14c97799c323/linux/x64/tar```

Reply:
Yeah. Updated here with a new OSD test manifest for 1.3.0.